zhaquirks/philips/hue_go.py

```diff
@@ -0,0 +1,12 @@
+"""Philips Hue Go device."""
+
+from zigpy.quirks.v2 import QuirkBuilder
+
+from zhaquirks.philips import PHILIPS, PhilipsHueCluster
+
+(
+    QuirkBuilder(PHILIPS, "7602031P7")
+    .also_applies_to(PHILIPS, "7602031U7")
```

Comment:
I've spotted a difference from your quirk builders, and it turns our `also_aplies_to` is just compatibility alias, so it could be changed.
```suggestion
    .applies_to(PHILIPS, "7602031U7")
```

source: https://github.com/zigpy/zigpy/blob/0f42cab8f7058cc68044678e4e6e46e59d1747b9/zigpy/quirks/v2/__init__.py#L554

Reply:
Good catch!